src/routes/blog/posts/announcing-files-v2-4-30/+page.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Introducing Files, version 2.4.30
+description: Better Home Page, Better Workflow
+thumbnail: /blog-resources/files-2-4-30/HeroImage.jpg
```

Comment:
you put a hypen between `files` and the version number which doesn't exist in the folder name itself.

```suggestion
thumbnail: /blog-resources/files2-4-30/HeroImage.jpg
```